.eslintrc.js

```diff
@@ -1,79 +1,142 @@
 module.exports = {
+  extends: ['./.eslintrc.base.js', 'plugin:prettier/recommended'],
+
```

Comment:
It may be a bit too late for that, as this PR has been written to follow the updated config and not the old config. It also includes all relevant changes to other files in the repo (choosing either to follow or ignore rules), such that merging this PR per se would not cause any new eslint errors or warnings.

I have left two somewhat related albeit now outdated comments higher up wrt to this PR, though they have been buried by later commit logs (requires expanding to view more). One of them talks a bit about eslint

Reply:
I think it might make sense for the eslint changes to be extracted to a separate PR that's based on the modules repo's master branch. The changes are quite substantial, so someone else might want to review them. Then that branch might have to be merged before this one